src/app/community/community.component.html

```diff
@@ -8,19 +8,29 @@ <h3 style="text-align: right; margin-right: 0.5rem;">
               <button mat-stroked-button (click)="openAddMessageDialog()" *ngIf="showNewsButton" i18n>New Voice</button>
             </ng-container>
           </h3>
-          <ng-container *ngIf="isLoading; else loaded">
+          <ng-container *ngIf="isLoadingInit; else loaded">
             <span i18n>Loading voices...</span>
           </ng-container>
           <ng-template #loaded>
             <ng-container *ngIf="news?.length > 0; else noVoices">
-              <planet-news-list [items]="news" [shareTarget]="shareTarget" (viewChange)="toggleShowButton($event)" [viewableId]="teamId"></planet-news-list>
+              <div class="news-container">
+                <planet-news-list [items]="paginatedNews" [shareTarget]="shareTarget" (viewChange)="toggleShowButton($event)" [viewableId]="teamId"></planet-news-list>
+              </div>
+              <div class="full-width" style="text-align: center;" *ngIf="!isLoadingInit && !isLoadingMore && hasMoreNews && !isActiveReplyThread">
```

Comment:
The function needs to be called for this to work properly (`!isactiveReplyThread()`).

I think it's not optimal for rendering reasons to call a function in an `ngIf`. Better to have a variable because Angular can detect when that changes and optimize when it runs a rerender.